### ISSUE TITLE
Indexing records into elasticsearch

### ISSUE THREAD

**Issue by jobrieniii** [540co/epa-rfi-backend#14]
@aaronbullard  - are there instructions on how to index / migrate records into elasticsearch?

**Comment by aaronbullard** [comment]
No. That wasn't within our original scope. It's certainly a feature that could be added.

**Comment by patricknowlan** [comment]
Create instructions in the readme on how to load data, run the transformation script, setup elasticsearch, and add the data to elastic. Or is there a way to add the json files or allow developers to get the json files to use in their own environment.
